### PR TITLE
No Jira: Minor housekeeping for HCP docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17

Issue: No Jira - this PR fixes incorrect links and removes an outdated TP boilerplate statement

Link to docs preview:
- [Gathering information to troubleshoot hosted control planes](https://82288--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-troubleshooting.html#hosted-control-planes-troubleshooting_hcp-troubleshooting) (updated link to point to OCP docs instead of RHACM docs)
- [Advanced node tuning for hosted clusters](https://82288--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/using-node-tuning-operator#advanced-node-tuning-hosted-cluster_node-tuning-operator) (updated link to point to OCP docs instead of RHACM docs)
- [Deploying the SR-IOV Operator for hosted control planes](https://82288--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-machine-config#sriov-operator-hosted-control-planes_hcp-machine-config) (removing link and removing outdated TP boilerplate)
- [Backing up and restoring etcd on AWS](https://82288--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp_high_availability/hcp-backup-restore-aws) (removing outdated TP boilerplate)
- [Disaster recovery for a hosted cluster in AWS](https://82288--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-aws) (removing outdated TP boilerplate)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by xenolinux — COMMENTED** [review]
LGTM

File: hosted_control_planes/hcp-troubleshooting.adoc [hunk 1]
```diff
@@ -9,6 +9,12 @@ toc::[]
 If you encounter issues with {hcp}, see the following information to guide you through troubleshooting.
 
 include::modules/hosted-control-planes-troubleshooting.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../hosted_control_planes/hcp-prepare/hcp-cli.adoc[Installing the {hcp} command line interface]
```
Comment (xenolinux) [inline hosted_control_planes/hcp-troubleshooting.adoc:16]:
```suggestion
* xref:../hosted_control_planes/hcp-prepare/hcp-cli.adoc[Installing the {hcp} command-line interface]
```

To match with the section title

File: modules/hosted-control-planes-troubleshooting.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ Although the output does not contain any secret objects from the cluster, it can
 
 * You need the `name` value for the `HostedCluster` resource and the namespace where the CR is deployed.
 
-* You must have the `hcp` command line interface installed. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-install-cli[Installing the {hcp} command line interface].
+* You must have the `hcp` command line interface installed. For more information, see "Installing the {hcp} command line interface".
```
Comment (xenolinux) [inline modules/hosted-control-planes-troubleshooting.adoc:32]:
```suggestion
* You must have the `hcp` command line interface installed. For more information, see "Installing the {hcp} command-line interface".
```

File: hosted_control_planes/hcp-troubleshooting.adoc (hunk 1 above)
Comment (lahinson) [reply hosted_control_planes/hcp-troubleshooting.adoc:16]: Will fix

**Pull request merged**